client/src/search.tsx

```diff
@@ -15,6 +15,10 @@ import { getCollectionItems } from "./plus/collections-quicksearch";
 const PRELOAD_WAIT_MS = 500;
 const SHOW_INDEXING_AFTER_MS = 500;
 
+function qsURL(url, input) {
+  return `${url}?qs=${input}`;
```

Comment:
Is it worth doing this programmatically using `URL`, or over-engineering?

```suggestion
  const urlObject = new URL(url);
  urlObject.searchParams.set('qs', input);
  return urlObject.toString();
```

Also, could we type the parameters as `string`?

Reply:
`url` is often just a path (ignore the misleading name) and therefore this would be more complex like:
```js
  const urlObject = new URL(url, "https://example.com);
  urlObject.searchParams.set('qs', input);
  const urlString urlObject.toString();
  return urlString.replace(/.*:\/\/[^\/]*/, "");
```